lambda/templates/av_lambda.json.tpl

```diff
@@ -20,7 +20,9 @@
       ],
       "Resource" : [
         "arn:aws:s3:::tdr-upload-files-dirty-${environment}/*",
```

Comment:
Maybe order alphabetically?

Reply:
I've changed them round. We should be able to remove the original permissions once we've switched things over to the new bucket.